packages/client/utils/draftjs/simpleDOMBuilder.ts

```diff
@@ -0,0 +1,14 @@
+import {JSDOM} from 'jsdom'
+
+function simpleDOMBuilder(html) {
```

Comment:
+1 `client/utils` seems like an odd place to put it if it's only needed on server side.

Reply:
yeah, make sense, and the `'jsdom'` only added to `parabol-server` packages.